### PR TITLE
LGA-1830 Alternative Help Face to Face

### PR DESCRIPTION
**Pull request by jogilder**
1. Updated step key words in alternative_help.feature so it reads better
3. Added in steps to select alternative help and go to the face to face tab.
4. Tech debt:
a) Removed repeated constant in `constants.py` and changed references in code. Moved code only used in these steps out of `common_steps.py`
b) Combined diagnosis features and steps into diagnosis.feature and `diagnosis.py`
c) added in find_by_link_text so can use it for generic function to find a hyperlink based on the text within it.
d) wrap a button click in helper function so that it doesn't fail on `staleexception`. Also changed to `find_by_xpath` as it doesn't matter if there is more than one.

### CODE REVIEW

File: behave/features/alternative_help.feature [hunk 1]
```diff
@@ -7,15 +7,18 @@ Background: Login
...
+    AND I select ‘Create Scope Diagnosis'
     AND I select the diagnosis <category> and click next <number> times
     | category                                                | number |
     | Crime                                                   | 2      |
-    THEN I get an "OUTOFSCOPE" decision
-    WHEN I select 'Assign Alternative Help'
-    THEN I am taken to the "Alternative help" page for the case located at "/alternative_help/"
+    AND I get an "OUTOFSCOPE" decision
+    #This is the icon in the top RH corner
+    AND I select 'Assign Alternative Help'
```
Comment (said-moj) [inline behave/features/alternative_help.feature:20]: What do you of changing the step to include it's for an icon

File: behave/features/diagnosis.feature [hunk 2]
```diff
@@ -21,13 +21,29 @@ Scenario: Given an INSCOPE decision when selecting debt and court proceedings
     And I get an "INSCOPE" decision
     And select 'Create financial assessment'
     Then I am taken to the Finances tab with the ‘Details’ sub-tab preselected
+    #This is the icon in the top RH corner
     When I select 'Assign Alternative Help'
```
Comment (said-moj) [inline behave/features/diagnosis.feature:25]: What do you of changing the step to include it's for an icon?

File: behave/features/steps/common_steps.py [hunk 3]
```diff
@@ -23,11 +22,28 @@ def assert_header_on_page(title, context):
 
 def wait_until_page_is_loaded(path, context):
     def do_test(*args):
+        # import pdb
```
Comment (said-moj) [inline behave/features/steps/common_steps.py:25]: Can we remove these debug steps?

File: behave/features/alternative_help.feature (hunk 1 above)
Comment (jogilder) [reply behave/features/alternative_help.feature:20]: Do you mean changing the step so it says `I select 'Assign Alternative Help' icon`?

File: behave/features/steps/common_steps.py (hunk 3 above)
Comment (jogilder) [reply behave/features/steps/common_steps.py:25]: yes

File: behave/features/diagnosis.feature (hunk 2 above)
Comment (jogilder) [reply behave/features/diagnosis.feature:25]: good idea, will do.

File: behave/features/alternative_help.feature (hunk 1 above)
Comment (said-moj) [reply behave/features/alternative_help.feature]: @jogilder yep

**Pull request merged**